### ISSUE TITLE
The connection will never Yield None

### ISSUE THREAD

**Issue by foxyblue** [revolut-engineering/data-libs#44]
On [this line](https://github.com/revolut-engineering/data-libs/blob/master/connections/revlibs/connections/connectors.py#L108-L113), the code, which yeilds `None` will never be reached. Since `obj()` is always instantiated.

```
    connector = obj(cfg)
    if connector:
        yield connector.connect()
        connector.close()
    else:
        yield None
```

Georgy, has suggested allowing this to yeild `None` if `connector.connect()` fails.

**Comment by gelioz** [comment]
Fixed in https://github.com/revolut-engineering/data-libs/pull/51

**Comment by foxyblue** [comment]
@gelioz Adding

closes: #44

in a PR will automatically close issues. :)
